pkgs/development/python-modules/result/default.nix

```
@@ -9,14 +9,14 @@
 
 buildPythonPackage rec {
   pname = "result";
-  version = "0.16.1";
+  version = "0.17.0";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
Could you replace deprecated `format = "setuptools"` with `pyproject = true;` and add `build-system` to prevent legacy build?
https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters

---

adjusted in https://github.com/NixOS/nixpkgs/pull/316909/commits/52ec0675e7463d44d8fe8909b317d8a5e478c3f2